pkg/eks/eks.go

```diff
@@ -104,7 +104,11 @@ func (c *ClusterProvider) doListClusters(chunkSize int64, printer printers.Outpu
 		if err != nil {
 			return err
 		}
-		allClusterNames = append(allClusterNames, clusters...)
+
+		for _, clusterName := range clusters {
+			allClusterNames = append(allClusterNames, &map[string]string{"name": *clusterName, "region": c.Spec.Region})
```

Comment:
Oops, I think I've misguided you on this one, I'm sorry... I was assuming something which now appears completely wrong, original version with a struct was actually much better. What I was thinking is that we can have map from names to regions, but that's very very shortsighted, and we's have to have a map of regions to slices of string or something like that. Let's just go back to slice of structs, pardon me that's I've not given it enough thought originally.

Reply:
That's OK, I'm happy to change it back. Although, I'll admit that I like the way it looks now 😆.